tooling/cli/src/interface/rust.rs

```diff
@@ -415,40 +415,43 @@ impl Rust {
     let tauri_path = tauri_dir();
     let workspace_path = get_workspace_dir()?;
 
-    let watch_folders = if tauri_path == workspace_path {
-      vec![tauri_path]
-    } else {
-      let cargo_settings = CargoSettings::load(&workspace_path)?;
-      cargo_settings
-        .workspace
-        .as_ref()
-        .map(|w| {
-          w.members
-            .clone()
-            .unwrap_or_default()
-            .into_iter()
-            .map(|p| workspace_path.join(p))
-            .collect()
-        })
-        .unwrap_or_else(|| vec![tauri_path])
-    };
+    // We always want to watch the main tauri folder.
+    let mut watch_folders = vec![tauri_path];
 
-    let watch_folders = watch_folders
-      .into_iter()
-      .flat_map(|p| {
-        match expand_member_path(&p) {
-          Ok(p) => p,
-          Err(err) => {
-            // If this fails cargo itself should fail too. But we still try to keep going with the unexpanded path.
-            error!("Error watching {}: {}", p.display(), err.to_string());
-            vec![p]
+    // We also try to watch workspace members, no matter if the tauri cargo project is the workspace root or a workspace member
```

Comment:
this logic now needs a function on its own IMO.

Reply:
Just the "get workspace member paths" part or the whole "get watch folders" block? It's only a one line difference so excuse the stupid question but i don't wanna do it twice :D

Reply:
hard to say tbh, just go with your gut, I am fine with either way